### ISSUE TITLE
Didn't work in Github Action V2

### ISSUE THREAD

**Issue by shiipou** [kentaro-m/add-an-issue-reference-action#3]
**Describe the bug**
The args JSON string have escaped quote disapear, so JSON parse is impossible.

**To Reproduce**
1. Create YAML file in .github/workflows/
2. Write the workflow
3. Create an issue, a branch named 'issue-X' where X is the number of the issue & a pull_request
4. See error in workflow log

or see [here](https://github.com/Nocturlab/projet-archi-info-server/tree/453c24c7b78b6da7ee18964040eb39f712b18149)

**Expected behavior**
This would create a comment in the pull_request to link issue with it.

**Screenshots**

Workflow file :
```
on: pull_request
name: Add an issue reference to they pull request
jobs:
  addAnIssueReference:
    name: Add an issue reference
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - name: Add an issue reference
      uses: kentaro-m/add-an-issue-reference-action@master
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        args: "{\"branch\":\"issue-\"}"
```

Workflow error log :
```bash
Run kentaro-m/add-an-issue-reference-action@master
/usr/bin/docker run --name bb815f8f63f8950c4c93bb3f06c398b0d2fd_18dcc1 --label 04bb81 --workdir /github/workspace --rm -e GITHUB_TOKEN -e INPUT_ARGS -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/projet-archi-info-server/projet-archi-info-server":"/github/workspace" 04bb81:5f8f63f8950c4c93bb3f06c398b0d2fd {"branch":"issue-"}
undefined:1
{branch:issue-}
 ^

SyntaxError: Unexpected token b in JSON at position 1
    at JSON.parse (<anonymous>)
    at new IssueReferenceCreator (/index.js:4:22)
    at Object.<anonymous> (/entrypoint.js:5:31)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
##[error]Docker run failed with exit code 1
```

**Comment by kentaro-m** [comment]
Hi, @shiipou. Thank you for reporting the problem.
Currently, codes of the master branch are written for GitHub Actions v1 and is deprecated.  I'm just working to migrate GitHub Actions from v1 to v2. https://github.com/kentaro-m/add-an-issue-reference-action/pull/4

**Comment by shiipou** [comment]
> Hi, @shiipou. Thank you for reporting the problem.
> Currently, codes of the master branch are written for GitHub Actions v1 and is deprecated. I'm just working to migrate GitHub Actions from v1 to v2. See the detail: #4

Thank you, that will help me

**Comment by kentaro-m** [comment]
@shiipou 
I merged PR #4 to migrate GitHub Actions from v1 to v2 and released the action of the new version. Please check it If you want.

**Comment by shiipou** [comment]
Thanks, I'll add it to my private repo to test it.
